src/AppBundle/Controller/DefaultController.php

```diff
@@ -116,13 +118,10 @@ private function getEventFeed()
                     $events[$i]['SUMMARY'] = stripslashes($event['SUMMARY']);
                 }
             }
-            else {
-                $events = array();
-            }
             $this->getCache()->save('mozilla_reps_events', serialize($events), 3600);
         }
 
-        return $events;
+        return $events ?? array();
```

Comment:
thanks a lot for your contribution! we could also initialize the $events array from the beginning instead of null so we don't need the check anymore :)